frameworks/g-cloud-12/questions/services/trainingServiceSpecificList.yml

```diff
@@ -11,9 +11,11 @@ type: list
 list_item_name: cloud hosting or software service
 
 validations:
+  - name: answer_required
+    message: Enter a service.
   - name: under_10_words
-    message: You can’t write more than 10 words for each cloud hosting or software service.
+    message: Each cloud hosting or software service must be 10 words or fewer.
   - name: max_items_limit
     message: You must have 10 or fewer cloud hosting or software services.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?